concent_api/concent_api/system_check.py

```diff
@@ -99,6 +99,14 @@ def create_error_22_if_download_leadin_time_has_wrong_value():
     )
 
 
+def create_error_23_if_concent_time_settings_have_wrong_value(concent_setting_name):
```

Comment:
I would consider splittimng this into two:
1) Not defined
2) Defined but wrong value
In `system_check` you make distinction between those two cases and it should be reflected in errors created